impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterConfig.java

```diff
@@ -516,7 +515,7 @@ private void configureKraftNode(KafkaEndpoints kafkaEndpoints, int nodeId, Prope
         putConfig(nodeConfiguration, "node.id", Integer.toString(nodeId)); // Required by Kafka 3.3 onwards.
 
         var quorumVoters = IntStream.range(0, kraftControllers)
-                .mapToObj(controllerId -> String.format("%d@//%s", controllerId, kafkaEndpoints.getEndpointPair(Listener.CONTROLLER, controllerId).connectAddress()))
+                .mapToObj(controllerId -> String.format("%d@%s", controllerId, kafkaEndpoints.getEndpointPair(Listener.CONTROLLER, controllerId).connectAddress()))
```

Comment:
Was the format wrong before or did it change as part of 3.8.0? I guess what I'm really getting at is does this need to checked per version somehow.

Reply:
The format used by the Ext for this property was wrong (according to the Kafka documentation).  The validation used by 3.8.0 rejects the slashes.